ClimsoftVer4/ClimsoftVer4/ucrHourly.vb

```diff
@@ -147,87 +118,52 @@ Public Class ucrHourly
             CallByName(fhRecord, kvpTemp.Value.Value.GetField(), CallType.Set, kvpTemp.Key.GetValue)
         Next
         ucrLinkedNavigation.UpdateNavigationByKeyControls()
+        'change the validation of the controls
         SetValueValidation()
     End Sub
+
+    Public Sub SaveRecord()
```

Comment:
I see you like having these subs here other than running them on the dialog, why is that?